### ISSUE TITLE
Negtive UVa and UVb readings in direct sunlight

### ISSUE THREAD

**Issue by sfletc** [arduino-libraries/Arduino_MKRENV#7]
I have 2 MKRENV shields.  Even when next to each other, one often provides negative UVa and UBb readings of down to ~ -700 in direct sunlight (the blue trace in the image, the green trace is
uva and uvb from two MKRENV shields from the MKRENV that is behaving normally).  Not sure if this is a library or a hardware issue, of the shield shouldn't be in direct sunlight.    

![uva and uvb from two MKRENV shields](https://user-images.githubusercontent.com/5491692/63657996-70c8e700-c7ea-11e9-9f24-c52206b8c7a8.PNG)

**Comment by sandeepmistry** [comment]
@Rocketct any updates on this?

**Comment by Rocketct** [comment]
yes i write a mail to vishay support(uv sensor manufacturer) and received a response, i have to test the the instruction and i 'll report the results

**Comment by sandeepmistry** [comment]
@Rocketct would it be possible to share their instructions here?

**Comment by Rocketct** [comment]
yes sure i'll attach pdf with the guidelines to calibrate the sensor for who want to test the calibration procedure, here folowing also the email by the support:

> The fact that you are getting negative values is all down to the calibration coefficients that you use during the UVI calculation. You will have seen in our application note ( attached for reference ) that there are 3 parts to this procedure.
> 1.  Calibrating the sensor such that the visible and IR leakage in the device is compensated for , by weighting the provided compensation channels respectively and subtracting these from the UVA and UVB output channels
> 2.   Calibrating a device , within enclosure , to a reference UVI meter under sunlight . This can be used as a golden sample.
> 3.  Calibrating each device to this golden sample under a reference light UVA/UVB light source , such that part to part variances are taken into account. 
> I have attached a separate PDF that gives an overview of this calibration procedure , also highlighting which coefficients come into play when.

> Furthermore, as also referenced in our application note , it is recommended that Teflon is used above the sensor , so that the optical angle response is cosine corrected (the optical response should correlate to a cosine curve). Without this , as the angular response(distribution of sensitivity over the angle) of the on chip filter does not natively have a cosine response, the position of the sun and the direction in which rays irradiate the sensor  will have a strong effect on the viability of the calculated UVI result. 



[DESIGNINGVEML6075_84339_20180605.pdf](https://github.com/arduino-libraries/Arduino_MKRENV/files/3631221/DESIGNINGVEML6075_84339_20180605.pdf)
